frontend/lib/laletterbuilder/faq-content.tsx

```diff
@@ -18,7 +18,8 @@ export const getFaqContent: () => FaqItem[] = () => [
         <Trans id="laletterbuilder.faq.whentosend">
```

Comment:
Do you need to change these ids? (I forget how this works)

Reply:
yeah, i changed them, ran yarn lingui:extract, then changed them back and ran it again so that the ids wouldn't have "v2" in them, i _believe_ it's just so that lingui knows to extract the updated string into the .po file so this should hopefully be fine..... it's also very unnecessary i just thought it looked a little nicer haha

Reply:
ah ok that makes sense! thanks